test/beast/http/field_compiles.cpp

```diff
@@ -0,0 +1,4 @@
+//
+// check that http/field.hpp compiles unaided
+//
+#include <boost/beast/http/field.hpp>
```

Comment:
`field.cpp` should accomplish this, no?

Reply:
See:
https://github.com/boostorg/beast/blob/83fc0018a89863c6c2aa3cf932313664f8651e9b/test/beast/http/field.cpp#L11

Reply:
no, because it pulls in the test stuff, which pulls in iostream.

I obviously tried this first.